addons/escoria-core/game/escoria.gd

```diff
@@ -124,6 +128,7 @@ func _init():
 func _ready():
 	settings = save_manager.load_settings()
 	apply_settings(settings)
+	room_manager.register_reserved_globals()
```

Comment:
This could be done in room_manager's `_init()` function (to create) since it is being done right after it was instanced.

Reply:
Are you sure? `ESCRoomManager` depends on/makes use of `ESCGlobalsManager`, which would mean we'd have to instantiate the two in order in the `_init` method which could produce issues if, for some reason, someone decides to re-order/refactor the method. Not to mention that `input_manager.register_core()` is also called in `ready`.

Let me know.

Reply:
Actually this would just be what you're doing here: 
L113 instanciates `room_manager`, and in `escoria._ready()` the function `room_manager.register_reserved_globals()` is called, so this happens every time Escoria is run. Since `register_reserved_globals()` is called nowhere else, I assume that can automated a bit.

Reply:
Ok I re-read your answer and your justification is satisfying :) Thanks!